### ISSUE TITLE
When non-essential icons are not properly hidden from screen reader users, they can be announced as unrecognized characters.

### ISSUE THREAD

**Issue by cryptasian** [microsoft/BotFramework-WebChat#4991]
### Is it an issue related to Adaptive Cards?

No.

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Live region, Transcript navigation

### What is the public URL for the website?

https://edfinancial.com

### How to reproduce the issue?

This is feedback from the accessibility company Allyant who has been testing our website and has people who use screenreaders and other assistive technologies as a part of their testing team.

![nonessential-icons](https://github.com/microsoft/BotFramework-WebChat/assets/30207740/191182d5-be7c-43c6-b097-9aa3c7712e42)

### What do you expect?

Emoji's are considered non-essential icons and should be hidden from screenreaders. 

Hide non-essential icons by moving them into their own <span>, or other inline tag, then giving the <span> an aria-hidden="true" attribute.

If a non-essential icon is already independently wrapped by an element, instead give this element an aria-hidden="true" attribute. Ensure that no other content is contained within the element wrapping the icon, as doing so may inadvertently or improperly hide the additional content from screen reader users.

### What actually happened?

Emoji's are not hidden from screenreaders.

### WCAG reference

https://www.w3.org/WAI/WCAG22/Understanding/non-text-content

**Comment by compulim** [comment]
The bot author is using emoji in the suggested actions. Thus, screen reader will read the emoji. This is expected.

Bot author should consider using images in the suggested actions. For details, please refer to https://github.com/Microsoft/botframework-sdk/blob/main/specs/botframework-activity/botframework-activity.md#image. Note, suggested action is a type of card action.
